cmd/daemon.go

```diff
@@ -97,7 +104,7 @@ func startDaemon() {
 		}
 		defer watcher.Close()
 
-		log.Println("Watching:", paths.GetSpotifyAppsPath(spotifyDataPath))
+		log.Println("Watcher: watching:", paths.GetSpotifyAppsPath(spotifyDataPath))
```

Comment:
maybe `[Watcher]`? Or perhaps leave it and I will make log function for every print/errors etc if you don't mind

Reply:
Let's not make this PR bigger than it needs to be, we can discuss how to do logging separately.